R/auth.R

```diff
@@ -0,0 +1,24 @@
+#' @title Get the API key
```

Comment:
note: [this page](https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html) says `@title` is optional unless you are doing something exotic.  The first line of the roxygen docstring should be inferred to be the title.  Same with `@description` below.  (Though `@description` may also be needed if you're adding an `@rdname`/`@describeIn` above it.)

question (non-blocking): Am I missing some benefit of explicit `@title` tags?  (Robust to auto-line-wrapping?)

Reply:
I think it was just suggested to me by auto-complete, I'll remove them.